client/lib/analytics/index.js

```diff
@@ -344,6 +354,31 @@ const analytics = {
 		}
 	},
 
+	// Lucky Orange tracking
+	luckyOrange: {
+		initialized: false,
```

Comment:
(You could then probably drop the `initialize` function in this file, and call `analytics.luckyOrange.addLuckyOrangeScript()` directly from `state/analytics/middleware#loadTrackingTool`.)